packages/analytics/amplify_analytics_pinpoint/example/tool/provision_integration_test_resources.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+
```

Comment:
request: Could this script take an `AWS_PROFILE` env variable like the other provision scripts? I believe `cdk` commands will take that param downstream.

Reply:
It looks like npm scripts can manage this so I've updated the scripts to use the AWS_PROFILE env variable